src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,7 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework18"/>
```

Comment:
Hello,
Your homework 18 has been reviewed, you're dogin great! 
But in the assignment  required add Homework18 class to the xml file and specify the `@Test method` to be ran in the xml file. But in the your code, I can't see define test method for xml file. Please add it.
//HINT: `Use the <methods> & <include> tags`
![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/144403954/7f78a54a-e5cd-47b0-9ac9-1edda228562c)